interface/modules/custom_modules/oe-module-changefeed/ModuleManagerListener.php

```
@@ -0,0 +1,146 @@
+
+        return $currentActionStatus;
+    }
+
+    /**
+     * @param string $modId
+     * @param string $currentActionStatus
+     */
+    private function unregister($modId, $currentActionStatus): mixed
+    {
+        return $currentActionStatus;
+    }
```

PHPMD (2.15.0)

[warning] Avoid unused private methods such as 'unregister'. (undefined)

(UnusedPrivateMethod)